sigle/src/modules/analytics/PublishedStoryItem.tsx

```
@@ -1,25 +1,53 @@
-import { ArrowRightIcon } from '@radix-ui/react-icons';
+import { ArrowLeftIcon, ArrowRightIcon } from '@radix-ui/react-icons';
 import { stagger } from 'motion';
 import { useMotionAnimate } from 'motion-hooks';
 import { useEffect } from 'react';
+import { useQuery } from 'react-query';
+import { sigleConfig } from '../../config';
 import { styled } from '../../stitches.config';
 import { SubsetStory } from '../../types';
 import { Box, Flex, Text } from '../../ui';
+import { AnalyticsHistoricalResponse } from './stats/types';
+import { FATHOM_MAX_FROM_DATE } from './stats/utils';
 
 const StoryImage = styled('img', {
   objectFit: 'cover',
   objectPosition: 'center',
   width: 60,
   height: 43,
   br: '$1',
-  ml: '$2',
 });
 
+const baseUrl = sigleConfig.baseUrl;
+
+const fetchStoryViews = async (storyId: string) => {
+  const url = `${baseUrl}/api/analytics/historical?dateFrom=${FATHOM_MAX_FROM_DATE}&dateGrouping=month&storyId=${storyId}`;
+
+  const statsRes = await fetch(url);
+  const statsData: AnalyticsHistoricalResponse = await statsRes.json();
```

could you please throw an error in case smth is wrong with fetch? :)